backend/cmd/server.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"flag"
+	"log"
+
+	"github.com/codescalersinternships/Flyspray/app"
+	"github.com/codescalersinternships/Flyspray/models"
+)
+
+func main() {
+	// take port number and db filepath as flags
+	var dbFilePath string
+	var port int
+
+	flag.StringVar(&dbFilePath, "d", "./flyspray.db", "Specify the filepath of sqlite database")
+	flag.IntVar(&port, "p", 8080, "Specify the port number")
+
+	flag.Parse()
+
+	// database client declaration
+	client := models.DBClient{}
```

Comment:
change the structure so the client takes the path and App doesn't call `start` also a client doesn't need start. `NewDBClient` can open the connection